docs/book/src/clusterctl/provider-contract.md

```diff
@@ -283,8 +283,6 @@ Provider authors should be aware of the following transformations that `clusterc
 * Enforcement of target namespace:
     * The name of the namespace object is set;
     * The namespace field of all the objects is set (with exception of cluster wide objects like e.g. ClusterRoles);
-    * ClusterRole and ClusterRoleBinding are renamed by adding a “${namespace}-“ prefix to the name; this change reduces the risks
```

Comment:
@wfernandes this is a clusterctl internal; we should probably re-consider this during the implementation phase of the CAPI provider operator